packages/graphile-build-pg/src/plugins/PgAllRows.js

```diff
@@ -57,6 +57,8 @@ export default (async function PgAllRows(
               primaryKeyConstraint.keyAttributeNums.map(
                 num => attributes.filter(attr => attr.num === num)[0]
               );
+            const isView = t => t.classKind === "v";
+            const hasUniqueId = a => !!a.find(elem => elem.name === "uniqueId");
```

Comment:
We should allow the snake-case version also. If both are present we should use the one with the smaller `num` value

Reply:
Good point, I'll add that.